src/MathProgBase.jl

```diff
@@ -4,6 +4,7 @@ module MathProgBase
 
 include(joinpath(dirname(@__FILE__),"SolverInterface","SolverInterface.jl"))
 using .SolverInterface
+using Compat
```

Comment:
> Among all the tags in `Compat` there is no commit which includes `issymmteric`

Why do you think that?

See https://github.com/JuliaLang/Compat.jl/blame/57122c4d8275f7ba41229c74f01db490311787aa/README.md#L192, so it was introduced in https://github.com/JuliaLang/Compat.jl/commit/3e53ba57e150229bc3590398b5040f2ff7c28fbc which you can see was first available in 0.7.13.